src/workerd/jsg/README.md

```diff
@@ -427,12 +452,6 @@ const func = getFunction();
 func(1);  // prints 1
 ```
 
-For the third case, because it is possible to create a `jsg::Function` from either a JS function
```

Comment:
If we're removing this, we should also update the section above where it says `generally three ways of using jsg::Function` right?

Reply:
Maybe? not sure. I want to go back and take another pass through this section.